react/components/EXPERIMENTAL_TableTree/README.md

```diff
@@ -131,7 +130,7 @@ function Example() {
     console.log(checkedItems)
```

Comment:
Useful?

Reply:
Yeah. It is just to illustrate the `onToggle` function (i found less annoying than `alert`).

Reply:
In the upcoming `full documentation PR` I'll create more meaningful examples.

Reply:
Didn't that it was in a readme. No problem yes.